addon/components/o-s-s/currency-input.ts

```diff
@@ -73,11 +74,24 @@ export default class OSSCurrencyInput extends Component<OSSCurrencyInputArgs> {
       'ArrowUp',
       'ArrowDown'
     ];
+
+    if (
+      (event.key === 'v' && (event.metaKey || event.ctrlKey)) ||
+      (event.key === 'c' && (event.metaKey || event.ctrlKey))
+    ) {
+      return;
```

Comment:
It's hard to test this part as, this is an early return to avoid the `event.preventDefault()`

Reply:
nit: `if (['c', 'v'].includes(event.key) && (event.metaKey || event.ctrlKey))`